### PR TITLE
Set up Guacamole Docker composition as a systemd service

### PR DESCRIPTION
**Pull request by dav3r**
Also, start it up after installation and enable it to start on boot.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
:+1:

File: files/guacamole-composition.service [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Docker composition for Guacamole
+After=docker.service network.target multi-user.target cloud-final.service
+Requires=docker.service
+
+[Service]
+Restart=always
+# Stop containers (if running) when unit is stopped
+ExecStartPre=/usr/local/bin/docker-compose -f /var/guacamole/docker-compose.yml down
```
Comment (felddy) [inline files/guacamole-composition.service]: Expand the `-f` to `--file`.

**Review by felddy — APPROVED** [review]
Looks good.  See my comment about `docker-compose` arguments.

File: files/guacamole-composition.service (hunk 1 above)
Comment (dav3r) [reply files/guacamole-composition.service]: Good catch- changed in https://github.com/cisagov/ansible-role-guacamole/pull/2/commits/e2913fa86a798121d17da3d0f629e21fdbbe4c0c

**Pull request merged**